src/accelerate/accelerator.py

```diff
@@ -217,6 +217,10 @@ class Accelerator:
             If set to `True`, in cases where the total batch size across all processes does not exactly divide the
             dataset, samples at the start of the dataset will be duplicated so the batch can be divided equally among
             all workers.
+        use_seedable_sampler (`bool`, *optional*, defaults to `False`):
+            Whether or not use a fully seedable random sampler ([`~data_loader.SeedableRandomSampler`]). Comes at a
+            cost of potentially different performances due to different shuffling algorithms, but will ensure the
+            training results are fully reproducible.
```

Comment:
Should it mention that users should manually `set_seed` if they enable this?

Reply:
Yes that would definitely be good. Will do so